spec/presenters/sort_presenter_spec.rb

```diff
@@ -0,0 +1,188 @@
+require 'spec_helper'
+require "helpers/taxonomy_spec_helper"
+
+RSpec.describe SortPresenter do
+  include GovukContentSchemaExamples
+
+  subject(:presenter) { described_class.new(content_item(sort_options: no_sort_options), {}) }
```

Comment:
Is this necessary (same as the one below which is used) ?

Reply:
No it's not needed, I ended up removing it in a later commit.

https://github.com/alphagov/finder-frontend/pull/1069/files#diff-7c61a92169a590942f2d8b5226c6937d